public/js/p3/widget/FilterContainerActionBar.js

```diff
@@ -810,6 +810,38 @@ define([
         let q;
         if (condition.type === 'str') {
           q = `${condition.op === 'NOT' ? 'ne' : 'eq'}(${condition.column},${condition.value})`
+        } else if (condition.type === 'date') {
+          const encode = (date) => {
+            if (!date) {
+              return '';
+            }
+
+            const parsedDate = new Date(date);
```

Comment:
I'm a little concerned that we may have data that could fail to parse and this is not handled here

Reply:
We currently don't really handle errors in advanced search and filters, which is on my list to do. We just ignore the input if it is not valid. eg; if user provides non-numeric values in numeric field, we ignore that and return 'full search'.

Not ideal but I intentionally left that part as it is for now so that nothing would happen in the page and user would have to update the input before moving forward. If 'date type' is not urgent, this PR can wait until I implement error handlings and displaying them to user properly.

Reply:
OK. Can you verify we won't get a page crash if there is invalid data ?

Reply:
Yes, I verify that on my local tests. And I will ask others to do an extensive test as well.